pytorch/vision
# [iOS] added VisionTestApp for testing vision ops

Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#3629 [iOS] added VisionTestApp for testing vision ops**

### How to build & run the test app via Xcode
- `cd` to `ios/VisionTestApp`
- run `./setup.sh` (this step requires your `wget` is pre-installed and make sure your Python torch and Python vision are up to date so that the model is able to be generated)
- open `VisionTestApp.xcodeproj` using Xcode
- build & run (supports both simulators and real devices)
- check if the benchmark results can be displayed on the view
- if you want to check in some code, make sure you run `./clean.sh` before commit so that the model file and static libs will not be committed
